content/en/docs/reference/scheduling/config.md

```diff
@@ -58,6 +59,9 @@ extension points:
    Policy and are used to filter out nodes that can not run the Pod. Filters
    are called in the configured order. A pod is marked as unschedulable if no
    nodes pass all the filters.
+1. `PostFilter`: These plugins are called in their configured order when no
+   feasible nodes were found for the pod. If any postFilter plugin marks the Pod
```

Comment:
Is it PostFilter or postFilter?

Reply:
`postFilter` in the context of the configuration file. `PostFilter` in the context of the code interface that plugins implement. So, for the purpose of this doc, it should be `postFilter`. Fixed all the other references.